drivers/sensor/max6675/max6675.c

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2021 Teslabs Engineering S.L.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT maxim_max6675
+
+#include <drivers/sensor.h>
+#include <drivers/spi.h>
+#include <sys/byteorder.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(max6675, CONFIG_SENSOR_LOG_LEVEL);
+
+/** Thermocouple input bit (for NC detection). */
```

Comment:
It would be nice to document that this is inverted logic from what's implied by the name: the bit is 1 if there is no thermocouple.

Reply:
done